docs/guidance/implementation.md

```diff
@@ -0,0 +1,51 @@
+# Implementation models
+
+OC4IDS implementation involves combining data on infrastructure projects and contracting processes. This guidance describes some examples of implementation models used by OC4IDS publishers. It is not an exhaustive list, but it can be used to inform your implementation.
+
+Sources of data can include infrastructure transparency portals, procurement systems and project administration systems. OC4IDS implementation can also involve using contracting data published in OCDS format.
+
+In this guidance we discuss options for the collection of data, the flow of data between systems and the publication of data. For more information about the design of the system architecture to support this process, see the [System Architectures](https://standard.open-contracting.org/latest/en/guidance/build/system_architectures/) guidance in the OCDS documentation.
+
+## Standalone infrastructure transparency portal
+
+![Standalone infrastructure transparency portal](../_static/images/implementation_scenario_1.png)
+
+In this model, procuring entities enter project and contract level data directly into an infrastructure transparency portal and the portal publishes project data and summary contracting process data in OC4IDS format. CoST Honduras uses this model in [SISOCS](https://sisocs.org/), as does CoST West Lombok in [INTRAS](http://intras.fllajlombokbaratkab.or.id/).
+
+The main benefit of this approach is that it is relatively simple because it does not involve integrating data from different systems. The downside of this approach is that it can increase the data entry burden on procuring entities; for example, if they also need to enter the data about contracting processes into a separate procurement system.
```

Comment:
```suggestion
The main benefit of this approach is that it is relatively simple because it does not involve integrating data from different systems. The downside of this approach is that it can increase the data entry burden on procuring entities who may also need to enter contracting process data into a separate procurement system.
```